virt/virt-4-8-release-notes.adoc

```diff
@@ -53,6 +53,9 @@ The SVVP Certification applies to:
 ** Storage
 ** Guest memory swapping
 
+//CNV-12346 Windows VMs will now only be scheduled to hyper-v capable nodes
+* Windows virtual machines can now only be scheduled to HyperV capable nodes.
```

Comment:
Is the diff between the two lines on purpose? Can vs. will.

Reply:
Otherwise looks good.

Reply:
Typically, we don't use "will" because our style guidelines say we should avoid future tense like "will" and "will not" unless absolutely necessary.

The first line is a commented out placeholder where I just copied the JIRA story text as it is. It doesn't actually appear in the release notes. The actual text that is in the release notes is the 2nd line.

Reply:
Thanks!
/lgtm

Reply:
cc @ksimon1

Reply:
Should we state that this is only applicable for Windows VMs created from templates or those with the relevant hyperV capabilities stated in the yaml?

Reply:
So would the correct way of saying this be "If a Windows virtual machine is created from a template or has predefined HyperV capabilities, it can now only be scheduled to HyperV capable nodes."? 

Also, is it HyperV, hyperV, Hyper-V, hyper-v, or what? I've seen it written multiple ways and am not sure which is the correct form.

Reply:
Yes, /lgtm.

It should be Hyper-V (@karels please keep me honest here)